governance/votes/Executive vote - August 20, 2021.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Template - [Executive Vote] MATIC and PAX PSM Onboarding, 6s Updates, and Reducing Vault Liquidation Ratios - August 20, 2021
```

Comment:
So it's getting harder to figure out what goes in title and what goes in description. It's a little confused here, because title is more specific for onboarding, but less for 6s. Vice versa for description. Probably it should be description is has more detail